### PR TITLE
Associate model + search categories + new filter

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/products-super-power/enrich-products-variants.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ If you are looking for clothing with the color "red" and the size "S", the red S
 
 ![Search results with color red and size S](../img/Products_ProductModel_Search3.png)
 
+:::info
+_Since 2.3_ if you search by categories, the grid will automatically display the product models or the products classified according to the category selected in the grid.
+If a product model is not classified in the selected category but its children products are, the products will be displayed in the grid.
+So all the filters, attributes and categories selected in the grid, will be taken into account to display the level you would like to enrich.
```
Comment (KmilleF) [inline content/md/products-super-power/enrich-products-variants.md]: Maybe we could add that, if the selected filters wcould display a product model and its variant, the product grid will always display the variant product.

Comment (Delphineray) [reply content/md/products-super-power/enrich-products-variants.md]: No, really it displays the upper level so the product model, it's my previous example on the red color, the sub product model is red, its variant products are red so we display the sub product model red, but I add some details.

File: content/md/products-super-power/enrich-products-variants.md [hunk 2]
```diff
@@ -38,25 +38,59 @@ If the background of the `Variant products` is:
 
 ## Filter on your attributes
 
-To easily manage the products with variants and their until 3 levels of enrichment, we have added a **smart** search in the grid for products with variants to automatically display what you are looking for.
+To easily manage the products with variants up to 3 levels of enrichment, we added a **smart** search in the grid for products with variants to automatically display what you are looking for. Either a product model, or a product.
```
Comment (KmilleF) [inline content/md/products-super-power/enrich-products-variants.md]: isn't it the otther way round: "to easily manage the up to 3 levels of enrichment of procdcts with variant"? Caill, we miss you!!!

File: content/md/products-super-power/enrich-products-variants.md [hunk 3]
```diff
@@ -38,25 +38,59 @@ If the background of the `Variant products` is:
...
+
+Let's take an example with a t-shirt model "model-tshirt-divided" available in 3 colors (blue, red, grey) and 4 different sizes (S, M, L, XL).
+When you open the grid, one line for this t-shirt model is displayed (12 variant products are grouped in this product model).
 
 ![Grid with product models](../img/Products_ProductModel_Search1.png)
 
-As done on e-commerce websites, if you are looking for clothing with the color "red", the red t-shirt will be automatically displayed in the grid.
+As done on e-commerce websites, if you are looking for clothes with the color "red", red clothes will be automatically displayed in the grid.
+
+So in our example, you add the filter on the attribute `color` and select the option `red`, the product model displayed in the grid is now the "model-tshirt-divided-red" (with 4 red variant products in S, M, L, XL sizes)
```
Comment (KmilleF) [inline content/md/products-super-power/enrich-products-variants.md]: "the prduct displayed in the grid"?

Comment (KmilleF) [reply content/md/products-super-power/enrich-products-variants.md]: unless color is at the product model level?

File: content/md/products-super-power/enrich-products-variants.md (hunk 2 above)
Comment (Delphineray) [reply content/md/products-super-power/enrich-products-variants.md]: Updated with the 2.0 release note sentence, validated by our teacher Camille ;)

File: content/md/products-super-power/enrich-products-variants.md (hunk 3 above)
Comment (Delphineray) [reply content/md/products-super-power/enrich-products-variants.md]: yes color is at the product model level, but I will change for sub product model with link :)

**Pull request merged**